cmp/relativetimestamp/RelativeTimestamp.ts

```diff
@@ -62,6 +63,16 @@ export interface RelativeTimestampOptions {
 
     /** Time to which the input timestamp is compared. */
     relativeTo?: Date | number;
+
+    /**
+     * Governs if calendar days should be used for computing string rather than the native
+     * moment 24-hour day. Set to 'limited' to ensure that any output that refers to 'days'
+     * (i.e. between 1 day and 26 days elapsed time) will refer to the difference in
+     * *calendar* days between the timestamps.   Set to 'full' to ensure that any output less
+     * than 25 days will refer to the difference in *calendar* days. Off (default)
+     * will indicate that the standard moment formatting should be used.
```

Comment:
I find this comment confusing - perhaps it needs more emphasis on the **difference** between full and limited (i.e. "In limited mode, output of less than a day will be left as is")

Reply:
totally agree